src/Node/Internal.hs

```diff
@@ -773,10 +774,10 @@ nodeDispatcher node handlerIn handlerInOut =
     connectionOpened
         :: DispatcherState peerData m
         -> NT.ConnectionId
-        -> NT.Reliability
+        -> NT.Reliability -- TODO: this is not actually used. If this is intentional, we should remove the parameter.
```

Comment:
You can go ahead and remove it if you like. It's certainly not used now, we'll put it back if we ever need it.